### PR TITLE
Use app.css instead of different CSS files

### PR DESCRIPTION
**Pull request by raghunayyar**
Generating a singular CSS file from grunt exactly like it works for the javascript.

@georgehrke

### CODE REVIEW

File: js/Gruntfile.js [hunk 1]
```diff
@@ -42,20 +42,27 @@ module.exports = function(grunt) {
...
-			dist: {
+			basic: {
 				src: [
 					'<%= meta.configJS %>*.js',
 					'<%= meta.buildJS %>/**/*.js'
 				],
 				dest: '<%= meta.productionJS %>app.js'
-			}
+			},
+            extras: {
```
Comment (tcitworld) [inline js/Gruntfile.js]: Seems to be an indentation issue here.

Comment (raghunayyar) [inline js/Gruntfile.js]: weird, phpstorm needs to be configured. I will add another commit.

Comment (tcitworld) [inline js/Gruntfile.js]: I've have a lot of phpstorm indentation issues too. ;-)

**Comment by raghunayyar** [comment]
Btw, the PR is updated and is good for review.

**Comment by georgehrke** [comment]
Can you move the existing css files to css/app/ ?

**Comment by georgehrke** [comment]
cc @tcitworld

File: templates/main.php [hunk 2]
```diff
@@ -24,12 +24,7 @@
 $styles = [
 	'../js/vendor/fullcalendar/dist/fullcalendar',
 	'../js/vendor/jquery-timepicker/jquery.ui.timepicker',
-	'globals',
-	'calendar',
-	'calendarlist',
-	'datepicker',
-	'eventdialog',
-	'settings'
+	'../css/public/app'
```
Comment (georgehrke) [inline templates/main.php]: I'm pretty sure you can just say `public/app`

Comment (raghunayyar) [inline templates/main.php]: oh lol.

**Comment by georgehrke** [comment]
and please fix your local git config
https://github.com/owncloud/calendar/pull/660/commits
😉

**Pull request merged**